### PR TITLE
Update run-a-node.md

### PR DESCRIPTION
**Pull request by MaKla89**
# Description of change

Added a first draft of the main page for the "Run a node" section

### CODE REVIEW

File: docs/participate/support-the-network/run-a-node.md [hunk 1]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
 
-## Heading 2 (Sub topic 2)
+By having more active nodes in the network it grows more robust and less centralised. On the personal side you may feel more secure when sending transaction-requests to your own node. Keep in mind that - in contrast to other DLT - there is no direct or indirect financial incentive to run your own node. (Hint at Mana?)
+
+
+## What kinds of node-software is currently available for IOTA?
+
+On the one hand there are currently different IOTA networks active: 
+
+- The primary `Mainnet` (you might have heard the term "Chrysalis" or "IOTA 1.5") where for example the real tradeable IOTA-token belong to
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:23]:
```suggestion
- The primary `Mainnet` (you might have heard the term "Chrysalis" or "IOTA 1.5") where for example the real tradable IOTA-token belong to
```

File: docs/participate/support-the-network/run-a-node.md [hunk 2]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+- The `DevNet` on which new and not fully tested features are developed and tested. This is intended to ultimately lead the way to "IOTA 2.0"
+- There are further internal and community-driven networks for more specific test-runs, but those are out of scope for now ;)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:25]: We should definitely add the Testnet here. Maybe also Comnet? ;)

File: docs/participate/support-the-network/run-a-node.md [hunk 3]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+
+On the other hand there different implementations of the IOTA node software:
+
+- `Bee`: This software is mainly developed by the IOTA Foundation developers and is written in Rust. It is currently available for the `Mainnet`.
+- `Hornet`: This software is a mainly community-driven implementation written in Go and is primarily meant to work with the `Mainnet`.
+- `GoShimmer`: This software is currently the only node-software available for participating in the 'IOTA DevNet 2.0'. It is also not able to use goShimmer for the Mainnet
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:31]:
```suggestion
- `GoShimmer`: This software is currently the only node-software available for participating in the 'IOTA DevNet 2.0'. It is also not able to use GoShimmer for the Mainnet
```

File: docs/participate/support-the-network/run-a-node.md [hunk 4]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+
+Thus if you want to run a Mainnet-node you can chose between `Bee` and `Hornet`
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:33]:
```suggestion
Thus if you want to run a Mainnet-node you can choose between `Bee` and `Hornet`
```

File: docs/participate/support-the-network/run-a-node.md [hunk 5]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+
+
+## How to setup an IOTA node?
+
+There are different approaches to install and start IOTA nodes. The most common ones are as follows:
+
+- Download pre-compiled binary files and start the node (Easy to start with but might require more steps later on for configuration)
+- Rund the node-software as a docker-container (Easy for users who know docker)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:41]:
```suggestion
- Run the node-software as a docker-container (Easy for users who know docker)
```

File: docs/participate/support-the-network/run-a-node.md [hunk 6]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+- Clone the corresponding git repository, build the software yourself and start the node (Slightly more complex, might require intense copy-pasting of commands from guides ;))
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:42]: Do we want to add the possibility for APT repositories?

File: docs/participate/support-the-network/run-a-node.md [hunk 7]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
 
 
-## Heading 2 (Sub topic 1)
+## What is a node?
 
-content 
+Nodes are the backbone of distributed networks. In the IOTA case nodes specifically accept requests from participants (e.g. transactions from wallets), verify transactions and communicate among each other in case of unclear situations. This ultimately leads to a consensus and thus to a consistent ledger state for all nodes in the network. 
 
-## Heading 2 (Sub topic 2)
+By having more active nodes in the network it grows more robust and less centralised. On the personal side you may feel more secure when sending transaction-requests to your own node. Keep in mind that - in contrast to other DLT - there is no direct or indirect financial incentive to run your own node. (Hint at Mana?)
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:16]: I think we shouldn't mention Mana in regard to financial incentives. We don't know if there ever will be a marked and imo the wiki should be mostly fact based :)

File: docs/participate/support-the-network/run-a-node.md [hunk 8]
```diff
@@ -4,18 +4,48 @@ description: Everything you need to know and learn to setup an IOTA Mainnet or D
...
+On the other hand there different implementations of the IOTA node software:
+
+- `Bee`: This software is mainly developed by the IOTA Foundation developers and is written in Rust. It is currently available for the `Mainnet`.
+- `Hornet`: This software is a mainly community-driven implementation written in Go and is primarily meant to work with the `Mainnet`.
+- `GoShimmer`: This software is currently the only node-software available for participating in the 'IOTA DevNet 2.0'. It is also not able to use goShimmer for the Mainnet
+
+Thus if you want to run a Mainnet-node you can chose between `Bee` and `Hornet`
+
+
+## How to setup an IOTA node?
```
Comment (Dr-Electron) [inline docs/participate/support-the-network/run-a-node.md:36]:
```suggestion
## How to set up an IOTA node?
```

I think as it is used as a verb it should be `set up` and not `setup`. Same in the description of this page. Could someone confirm? 😅

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
Thanks for your contribution. I found some typos and we should talk about how we want to add the links to the tutorials. (Do we only want official guides at this point?)

File: docs/participate/support-the-network/run-a-node.md (hunk 7 above)
Comment (MaKla89) [reply docs/participate/support-the-network/run-a-node.md:16]: Good point, I'm perfectly fine with removing that :)

File: docs/participate/support-the-network/run-a-node.md (hunk 6 above)
Comment (MaKla89) [reply docs/participate/support-the-network/run-a-node.md:42]: I was also thinking about this, but so far this is only valid for Hornet, right? :/

**Review by MaKla89 — COMMENTED** [review]
All changes accepted. I'm not sure about the apt-installation as it is not available for all implementation. But in general I'd be in favor of adding apt-installation, yes :)

**Comment by Phyloiota** [comment]
> Thanks for your contribution. I found some typos and we should talk about how we want to add the links to the tutorials. (Do we only want official guides at this point?)

The linking issue is discussed internally in IF atm... Waiting for a feedback from Valerie or Navin on that

File: docs/participate/support-the-network/run-a-node.md (hunk 8 above)
Comment (Phyloiota) [reply docs/participate/support-the-network/run-a-node.md:36]: confirmed!

**Comment by Phyloiota** [comment]
> All changes accepted. I'm not sure about the apt-installation as it is not available for all implementation. But in general I'd be in favor of adding apt-installation, yes :)

I am very much for adding apt. Dont forget this Wiki should cover all available information. So people that want to use apt should find the information here

**Comment by MaKla89** [comment]
> I am very much for adding apt. Dont forget this Wiki should cover all available information. So people that want to use apt should find the information here

Yes, I know got a bit more of an idea about what this wiki should be about at its core and yes, apt should be added (I just dont know how to add it now in this stage of the pull request / review^^")

**Comment by Phyloiota** [comment]
> > I am very much for adding apt. Dont forget this Wiki should cover all available information. So people that want to use apt should find the information here
> 
> Yes, I know got a bit more of an idea about what this wiki should be about at its core and yes, apt should be added (I just dont know how to add it now in this stage of the pull request / review^^")

Ah cool, you can just continue to edit the file, this will create a new commit with the changes. But if you like we can do a call and i can lead you with screen sharing. Hit me up in discord!

**Comment by Dr-Electron** [comment]
Could you change this pull request to Draft?

**Review by Phyloiota — COMMENTED** [review]
I will close this, i have merged most of the content with the "become a node operator page" so we will not need double pages.

**Comment by Phyloiota** [comment]
closing, merged and added content with "become a node operator"

**Pull request closed without merging**